src/sentry/static/sentry/app/views/settings/projectAlerts/alertTypeChooser.tsx

```diff
@@ -9,16 +9,29 @@ import textStyles from 'app/styles/text';
 import BulletList from 'app/styles/bulletList';
 import FeatureBadge from 'app/components/featureBadge';
 import Tooltip from 'app/components/tooltip';
+import Feature from 'app/components/acl/feature';
 
 type Props = {
   selected?: string | null;
   onChange: (type: string) => void;
 };
 
+const MetricsTooltip = ({children}: {children?: React.ReactNode}) => (
+  <Tooltip
+    title={t(
+      `A metric is the value of an aggregate function like count() or avg()
+       applied to your events over time`
+    )}
+  >
+    <abbr>{children}</abbr>
+  </Tooltip>
+);
+
 const IssuesTooltip = ({children}: {children?: React.ReactNode}) => (
   <Tooltip
     title={t(
-      `An Issue is a unique error in Sentry, created by grouping error events based on stack trace and other factors.`
+      `An Issue is a unique error in Sentry, created by grouping error events
```

Comment:
This should change to what we have now with the flow builder: "Sentry groups similar events into Issues based on their stack trace and other factors"